lab-judy/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "lab-judy",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "./node_modules/webpack/bin/webpack.js",
+    "build-watch": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot",
+    "lint": "./node_modules/eslint/bin/eslint.js  ./",
```

Comment:
good work making a lint script, but you didn't include eslint in your package.json, so this script will not work for anyone pulling down your code :)